components/google_sheets/sources/new-worksheet-polling/new-worksheet-polling.mjs

```diff
@@ -0,0 +1,53 @@
+import googleSheets from "../../google_sheets.app.mjs";
+import common from "../common/new-worksheet.mjs";
+import base from "../common/http-based/base.mjs";
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+import sampleEmit from "./test-event.mjs";
+
+export default {
+  ...common,
+  key: "google_sheets-new-worksheet-polling",
+  name: "New Worksheet (Polling)",
+  description: "Emit new event each time a new worksheet is created in a spreadsheet.",
+  version: "0.0.1",
+  dedupe: "unique",
+  type: "source",
+  hooks: {
+    ...common.hooks,
+  },
+  props: {
+    googleSheets,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      static: {
+        intervalSeconds: DEFAULT_POLLING_SOURCE_TIMER_INTERVAL,
+      },
```

Comment:
This could use `default` instead of `static` so that users can adjust the interval.
```suggestion
      default: {
        intervalSeconds: DEFAULT_POLLING_SOURCE_TIMER_INTERVAL,
      },
```